<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_2</class>
 <widget class="QWidget" name="widget_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bobotin - Home</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/asset/Background.png);</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1202</width>
     <height>814</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="14" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>1200</width>
        <height>800</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1200</width>
        <height>800</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QLabel" name="tvTitle">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>1031</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>SF Pro Display</family>
         <pointsize>54</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #ffffff;
background: transparent;</string>
       </property>
       <property name="text">
        <string>BOBOTIN</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="edt_age">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>390</y>
         <width>411</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>SF Pro Display</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Masukan berat badan anda&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #FFFFFF;
border-radius: 24px;
padding-left: 14px;</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string>Umur</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="edt_height">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>460</y>
         <width>411</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>SF Pro Display</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Masukan tinggi anda&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #FFFFFF;
border-radius: 24px;
padding-left: 14px;</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string>Tinggi (cm)</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSubmit">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>640</y>
         <width>411</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>SF Pro Display</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">* {
	background: #686DE0;
	border-radius: 24;
	color: #FFFFFF;	
	padding: 12px;
}

*:hover {
	background: #6350E8;
}</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="text">
        <string>SUBMIT</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edt_weight">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>530</y>
         <width>411</width>
         <height>51</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>SF Pro Display</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Masukan umur anda&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #FFFFFF;
border-radius: 24px;
padding-left: 14px;
</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string>Berat (kg)</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="btnMale">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>330</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #130F40;
color: #130F40;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="btnFemale">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>330</y>
         <width>71</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #130F40;
color: #130F40;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="imgMale">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>190</y>
         <width>121</width>
         <height>121</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Laki-Laki&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 60px;
background-image: url(:/asset/boy-6.png);
background-repeat: no-repeat;
background-attachment: fixed;
background-position: center;
background-size: 15px 15px;
border: 4px solid #686de0;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="imgFemale">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>190</y>
         <width>121</width>
         <height>121</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Perempuan&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 60px;
background-image: url(:/asset/girl-1.png);
background-repeat: no-repeat;
background-attachment: fixed;
background-position: center;
background-size: 15px 15px;
border: 4px solid #686de0;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../asset.qrc"/>
 </resources>
 <connections/>
</ui>
